.env.tpl

```diff
@@ -13,3 +13,9 @@ NEXT_PUBLIC_SENTRY_DSN=https://bf79c216fe3c72328219f04aabeebc99@o609598.ingest.u
 NEXT_PUBLIC_SENTRY_ORG=storacha-it
 NEXT_PUBLIC_SENTRY_PROJECT=bluesky-backup
 NEXT_PUBLIC_SENTRY_ENV=development
+
+PRIVATE_KEY_1='{"kty":"RSA","n":"nfwYWpVTWS0yKkUTdDhrv9DSptmNJ1zC80RZR3tqAcRxSIlDxAMC9J7W1tnON6MiOK8oshifBPzi5l1AMXRAzYrvGM9mimxdKJx0ZJTSLIm0gIHw8-DRXHajQoMo8_Gs0F7RZ2a3RDrwn2x7Dt4aPaec8GMPFas866q-ne4WPSugZv38Dzb8hmHR1NSYqOYWc2S_EmEw_XCF7SGQKno4Ud6hePqLDPW4OTLH80c8czX_9jMHiL-YnxPk5Z8X3iO_xeu-nD__kqUSgBepDutAZ-0RxBAVRv6aYQIWV_gA2q1w9ACMK-jsblyXPx-DPfhvOb3w1DJxJmxSUYoq_4AM1w","e":"AQAB","d":"CYuvuFrDwvnL8_7heokQMEerUAveugKiVYrghBpXtjdK4FGe58CGKgamP4FrMt8R_xp3my8z_dv-emHRsktDwRzvGRuAUXEpEyjvkBuSiWthSHnRd8LdPjjS4NQbojBiOzgyGeJ8b0x1DoTs-IyAnw9kwvzmmUNpS79bg3UHy0v3pbOWXFEPzVfcpzqw5Uo6hAGY8iVkWO9uAhjdVJwl3WBOlt9Wr4CCxlqDU1xSlvU3RpdPp-evC8hhldYtxj7k7jXN5k2vYoLOm1jfdsM5h42S4xt0V1tUyHnK5ij2TbV0OgkaffDUdwipUiUOslVY6BmkxGTRt5akGl_GoaqZfQ","p":"2Vtzd52LcSm5e7vM_Y-EqUZlrT9zMVWVLwiJ5Pjo_2_aI6l24C0EqOXstTOCZI-_nMHoXHNhrApkHNmgRbp1vY_lTgpfr-ofv5R9iOKZrgts7P1ewleSOJQ_kAz6H2KH1gTDV23gMiRecVsAY88XZwmiCMbgdyYAmaJRWBcIxW0","q":"uhJuvxvMqJxvJMD3TmIinF2CHdrAfy_38IetofCNmn2mPDOeEqsnViV-iGAs7oTnbt1Hbj7RQCyjc-cTDIL-eYZu_e3YUJgDF15SxqWT2b-XHpnpa9ZfFmusZq67m14m7IXA3MB90wEtyEUisxPOmitPmiAz9h4t6kp8HM5KJNM","dp":"Bt5b9MC2wlm2MdHKdYzOfWbrGaFyq3Fv6G-Vo_W1rzTCqPPW_NiPWCMD6un4yeK9BGAgTqOax4wAg-TgqrNsuxPMBD5RrcZxQGWZ3fSDNLvJsmeDUhLgXQdaRtwSH2HGtx2pap7fQdZiYaSli4Hk1G26gBqAx2MdC73UYjtAHLU","dq":"OQ_iCpV549Nlej37GXya5J0AhCtLkTWisY7KPrlIk7IowpmJxmvael6xOR1Vaigz2RnfB_4XqbiGv8qDWvjpFVeWviSsi7qyZ_4zIFwyxTouG-JTZRCuxG9sxWice_8ZrLsv9ALME2NvKNWa2JQgbcMOfFOX28hRiGIjQao5rqU","qi":"uXOVv8V0gd95E5_VJ5KxvtJQyl19fziLbIEZcGIha-rC6WIq59edEWCMDfzQscDrhkqXILvB5CGmMtqXlKs2F1le7tonS4EMUPXuWIC3klTy8yyt4jvsHfNUmfCCHsU0sPnK33P_RzEVYPZGUByPcj8VyadLd_jO9-mDZHMFQjU"}'
```

Comment:
ruh roh - definitely shouldn't check this in, hope this wasn't too sensitive!

Reply:
This is environment variable for jwk private key which can be created in `scripts/generate-jwk.js`